godotenv.go

```diff
@@ -308,6 +318,16 @@ func parseValue(value string, envMap map[string]string) string {
 		}
 	}
 
+	// embed commands
+	if shouldParseEmbed {
+		if strings.HasPrefix(value, "$(") && strings.HasSuffix(value, ")") {
+			cmdStr := value[2 : len(value)-1]
+			cmd := exec.Command("/bin/sh", "-c", cmdStr)
```

Comment:
Shouldn't you do a check for the OS here and if on Windows create a PowerShell session?